src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -171,8 +173,8 @@ const UserSelect: FC<UserSelectProps> = ({
 
           //   return triggerRef.querySelector(`.${LABEL_CLASSNAME}`);
           // }}
-          trigger={!isUserAddressValid ? 'hover' : undefined}
-          placement={isUserAddressValid ? 'top' : 'bottom'}
+          trigger={isUserAddressInvalid ? 'hover' : undefined}
+          placement={isUserAddressInvalid ? 'bottom' : 'top'}
```

Comment:
Feels a bit weird to me to have this one specific tooltip placed below, but I've asked Mel to clarify whether this is necessary or not. Will add a comment here.

Reply:
Just for reference, the only other place I could find this tooltip being used:

<img width="517" alt="Screenshot 2024-06-20 at 15 30 20" src="https://github.com/JoinColony/colonyCDapp/assets/34915414/e103b8d2-26c2-48b9-a9bf-7556f8656577">

Reply:
> Feels a bit weird to me to have this one specific tooltip placed below, but I've asked Mel to clarify whether this is necessary or not. Will add a comment here.

Confirmation from Mel that all tooltips can also be positioned above, so this ternary can be removed. :)

Reply:
thanks! updated code according to this comments :) 

<img width="300" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/15706494/5eb34f0f-db48-4831-82ba-6c2df87fca94">